src/Pages/CreateQuestion.tsx

```diff
@@ -43,3 +105,26 @@ const CreateQBox = styled.div`
   gap: 30px;
   margin: 30px 0 30px 0;
 `
+
+const QuestionLabel = styled.div`
+  color: #3d8add;
```

Comment:
I see a lot of UI changes although this PR should focus on connecting API only. Some of the UI changes seem even outdated. Could you check UI changes again?

Reply:
> I see a lot of UI changes although this PR should focus on connecting API only. Some of the UI changes seem even outdated. Could you check UI changes again?

I changed the UI a bit to make getting information for the API easier; I should have created another pull request for it tho. I have adjusted it to make it up to date now :)